src/main/java/uk/gov/hmcts/reform/sscs/messaging/HmcMessage.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sscs.messaging;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
You probably want to provide an AllArgs and a NoArg constructor

Reply:
Would also put in model package